File: PhpFucntions/memory.php
<?php

function main($args){
    $number = $args["number"] ?? 10000000;
    $start = time();

    $result = [];
    for($i = 0; $i < $number; $i++)
        $result[$i] = $i;

    $end = time();
    $time = $end - $start;


    return ["time" => $time, "result" => count($result)];
}

File: PythonFunctions/memory.py
import time


def main(args):
    start = time.time()

    number = args.get("number", 100000)

    result = []
    for i in range(number):
        result.append(i)

    end = time.time()
    t = end - start
    return {"time": t, "result": len(result)}

